From 1e0f62acaacb4e7639d6203a9f008c66be712026 Mon Sep 17 00:00:00 2001
From: Serge Hallyn <serge.hallyn@ubuntu.com>
Date: Wed, 9 Oct 2013 13:01:18 -0500
Subject: [PATCH 1/1] ubuntu-cloud-prep hook: fix debug helper to not
 inappropriately fail

Bug found by Vincent Ladeuil <vila+ci@canonical.com>
Fix suggested by Scott Moser <smoser@ubuntu.com>

Signed-off-by: Serge Hallyn <serge.hallyn@ubuntu.com>
---
 hooks/ubuntu-cloud-prep | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hooks/ubuntu-cloud-prep b/hooks/ubuntu-cloud-prep
index f0b30ea..b3cf232 100755
--- a/hooks/ubuntu-cloud-prep
+++ b/hooks/ubuntu-cloud-prep
@@ -16,7 +16,7 @@ set -f
 VERBOSITY="0"
 
 error() { echo "$@" 1>&2; }
-debug() { [ "$VERBOSITY" -ge "$1" ] || return; shift; error "$@"; }
+debug() { [ "$VERBOSITY" -ge "$1" ] || return 0; shift; error "$@"; }
 fail() { [ $# -eq 0 ] || error "$@"; exit 1; }
 
 prep_usage() {
-- 
1.8.1.2

